bundles/AdminBundle/Resources/public/js/pimcore/asset/unknown.js

```diff
@@ -72,14 +72,22 @@ pimcore.asset.unknown = Class.create(pimcore.asset.asset, {
             items.push(this.workflows.getLayout());
         }
 
+        var iconClass;
```

Comment:
That shouldn't be necessary, as this is one already here: 
https://github.com/pimcore/pimcore/blob/3b20298ddae3b9c905fdc9db3582c9098a50c363/bundles/AdminBundle/Resources/public/js/pimcore/asset/asset.js#L82
So in my opinion this file doesn't need any modification.